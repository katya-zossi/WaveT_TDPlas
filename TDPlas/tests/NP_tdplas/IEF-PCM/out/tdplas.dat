 GEPOL-GB: COUNTING TESSERAE
 GEPOL-GB: GENERATING TESSERAE
 WARNING: You asked to print 0 plasmon modes. Number of plasmon modes to printchanged to          480
 Essential output written.
 tdplas calculation: FREQ and PROPAGATION namelist are n used
 Quantum Nanoparticle as external medium
 Local field effects are NOT included
 Medium polarization described by apparent charges
 This is a Quantum BEM run
 This run just writes matrices and boundary
 Medium polarization frozen to molecule in its GS
 Building surface from spheres.
 Cavity tessere number         480
 Drude-Lorentz model for dielectric function is used
 Used keyword: eps_A:   0.11022400000000000      , eps_gm:    7.5757600000000000E-003 , eps_w0:    0.0000000000000000      , f_vel:    0.0000000000000000     
 all modes diagonalized and printed 
 charges for MOPAC2002 interface PRINTED
Done reading input, took     0.004s
 Created output file with surface points
 Matrixes S D have been written out
