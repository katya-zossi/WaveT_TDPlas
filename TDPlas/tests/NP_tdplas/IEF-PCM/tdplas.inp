&tdplas 
/

&MEDIUM
medium_type='nan'     !nan: NP as external medium,  sol,q(uantum)so(l),q(uantum)nan(oparticle)
medium_pol='chr'      !chr: use apparent polarization charge, other option:dip(ole)
bem_type='diag'       !diag: use diagonal IEF, no other option working currently
bem_read_write='wri'  ! wri: just write matrices and boundary; other options: rea(d) full run reading matrix and boundary
/

&SURFACE
input_surface='bui' !gms:Surface read from file cavity.inp;  fil(e): read from file cavity.inp; bui(ld): build from spheres, to be given
spheres_number=2
sphere_position_x= 50. 140. !all in bohr
sphere_position_y=  0.   0.
sphere_position_z=  0.   0. 
sphere_radius=     50.  25.       
/

&EPS_FUNCTION
epsilon_omega='drl' !drl: Drude Lorentz, eps_d+eps_A/(w^2-eps_w0^2+i*eps_gm*w); deb: Debye
eps_0=1000.
eps_d=1. 
eps_A=0.110224   
eps_gm=0.00757576
eps_w0=0.  
f_vel=0. !Fermi velocity, to correct eps_gm for the limited mean free path
/

