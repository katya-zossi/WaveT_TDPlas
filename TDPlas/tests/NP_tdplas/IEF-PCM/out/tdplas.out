 Nanoparticle as external medium
 Medium polarization described by apparent charges
 Diagonal BEM furmulation
 This is a Classical BEM run
 This run just writes matrices and boundary
 Building surface from spheres.
 Number of spheres           2
 I_sphere  X     Y   Z   Radius
     1     94.4863      0.0000      0.0000     94.4863
     2    264.5616      0.0000      0.0000     47.2431
 Essential output written.
Done reading input, took     0.001s
 GEPOL-GB: COUNTING TESSERAE
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 nts_act         480
 GEPOL-GB: GENERATING TESSERAE
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo
 prima
 dopo

 TOTAL NUMBER OF SPHERES=    2
 SPHERE             CENTER  (X,Y,Z) (A)               RADIUS (A)      AREA(A*A)
   1   50.000000000    0.000000000    0.000000000   50.000000000
   2  140.000000000    0.000000000    0.000000000   25.000000000

 TOTAL NUMBER OF TESSERAE=     480
 SURFACE AREA=      39269.90816987(A**2)    CAVITY VOLUME=     589048.62254927 (A**3)
 nts_act         480
 Created output file with surface points
 Matrixes S D have been written out
